### ISSUE TITLE
Cmd + K to add link to a Button block focuses the browser address bar in Firefox

### ISSUE THREAD

**Issue by talldan** [WordPress/gutenberg#21777]
**Describe the bug**
Using a shortcut to add a link to the button block doesn't work in Firefox. Furthermore, multiple link popovers can be opened for multiple buttons (though this problem will likely go away when the first issue is fixed).

**To reproduce**
1. Create a post in Firefox (Tested on Mac OS X)
2. Add a Buttons block
3. Press Cmd+K to add a link to the block
4. Observe that the browser address or search bar is focused prompting the user to search
5. Add another button to the buttons block
6. Press Cmd+K again.
7. Observe multiple overlapping LinkControl popovers

**Expected behavior**
The link popover is focused when using the shortcut (`preventDefault`).

Clicking outside an opened popover closes it.

**Comment by afercia** [comment]
For me, macOS Firefox in English, Cmd+K focuses the Firefox Search instead (the one on the right of the address bar). I think the standard Firefox shortcut to focus the address bar is Cmd+L (L as "Location"), at least for English.

**Comment by talldan** [comment]
@afercia Good point. I think it's possible to have Firefox setup to either have separate search and address bars or a single unified bar. I have the latter configured in my Firefox.

**Comment by afercia** [comment]
Ah good point. Maybe new installations get the unified bar? I guess old installations that get updated retain old settings.

**Comment by afercia** [comment]
For reference: here's the related Firefox preference:

<img width="767" alt="Screenshot 2020-04-23 at 08 44 30" src="https://user-images.githubusercontent.com/1682452/80067749-c05cdf80-853e-11ea-879b-a913058b24c6.png">

And yes, when "Use the address bar for search and navigation" is on, Cmd+K moves focus to the unified address/search bar. 

Regardless, wouldn't this be solved with a simple `preventDefault()`?

**Comment by kirilzh** [comment]
I traced the bug down to https://github.com/WordPress/gutenberg/blob/f5b0c429ad7a535544bd013f665a74a79606f83d/packages/block-library/src/button/edit.js#L113

Currently there doesn't seem to be a way to preventDefault in `<KeyboardShortcuts />`

**Comment by talldan** [comment]
I think that would be handled in the `openLinkControl` event handler, @kirilzh.
